### PR TITLE
123 post page

### PR DESCRIPTION
**Pull request by onurccn**
added post page with hero animation
re-generated android and ios codes

### CODE REVIEW

**Review by YasirNacak — APPROVED** [review]

**Review by aErgani — APPROVED** [review]

File: sample/lib/models/view_models/post_view_arguments.dart [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import '../post.dart';
```
Comment (gurolcay) [inline sample/lib/models/view_models/post_view_arguments.dart]: package olarak alalim

File: sample/lib/ui/pages/post_page.dart [hunk 2]
```diff
@@ -0,0 +1,65 @@
+import 'package:flutter/material.dart';
+
+import '../../models/post.dart';
```
Comment (gurolcay) [inline sample/lib/ui/pages/post_page.dart]: package olarak alalim

File: sample/lib/ui/routes.dart [hunk 3]
```diff
@@ -5,15 +5,24 @@ import 'package:GTUBT/ui/pages/main_page.dart';
 import 'package:GTUBT/ui/pages/profile_page.dart';
 import 'package:GTUBT/ui/pages/signup_page.dart';
 import 'package:flutter/material.dart';
+import 'pages/main_page.dart';
```
Comment (gurolcay) [inline sample/lib/ui/routes.dart]: package olarak alalim

File: sample/lib/ui/pages/home_page.dart [hunk 4]
```diff
@@ -1,38 +1,56 @@
 import 'package:GTUBT/models/post.dart';
+import 'package:GTUBT/models/view_models/post_view_arguments.dart';
 import 'package:flutter/material.dart';
 
+import '../routes.dart';
```
Comment (gurolcay) [inline sample/lib/ui/pages/home_page.dart]: package olarak alalim

File: sample/lib/ui/pages/home_page.dart [hunk 5]
```diff
@@ -1,38 +1,56 @@
...
       title: "title5",
       summary:
           "text5 LOREM IPSUM DOLOR SIT AMET LOREM IPSUM DOLOR SIT AMET LOREM IPSUM DOLOR SIT AMET LOREM IPSUM DOLOR SIT AMET",
+      body:
+      "text4 LOREM IPSUM DOLOR SIT AMET LOREM IPSUM DOLOR SIT AMET LOREM IPSUM DOLOR SIT AMET LOREM IPSUM DOLOR SIT AMET",
       coverImageUrl: "https://via.placeholder.com/350x120.png",
       startDate: "02-05-2020");
 
+  void postTapEvent(BuildContext context, Post post, String heroTag) {
+    print(post);
```
Comment (gurolcay) [inline sample/lib/ui/pages/home_page.dart]: print ?

**Review by gurolcay — APPROVED** [review]

**Pull request merged**